libgraph/src/PropertyGraph.cpp

```diff
@@ -429,19 +430,21 @@ katana::PropertyGraph::DoWrite(
       handle, command_line, versioning_action,
       std::move(node_entity_type_id_array_res),
       std::move(edge_entity_type_id_array_res), node_entity_type_manager(),
-      edge_entity_type_manager());
+      edge_entity_type_manager(), txn_ctx);
 }
 
 katana::Result<void>
 katana::PropertyGraph::ConductWriteOp(
     const std::string& uri, const std::string& command_line,
-    katana::RDG::RDGVersioningPolicy versioning_action) {
+    katana::RDG::RDGVersioningPolicy versioning_action,
+    katana::TxnContext* txn_ctx) {
   katana::RDGManifest manifest = KATANA_CHECKED(katana::FindManifest(uri));
   katana::RDGHandle rdg_handle =
       KATANA_CHECKED(katana::Open(std::move(manifest), katana::kReadWrite));
   auto new_file = std::make_unique<katana::RDGFile>(rdg_handle);
 
-  if (auto res = DoWrite(*new_file, command_line, versioning_action); !res) {
+  if (auto res = DoWrite(*new_file, command_line, versioning_action, txn_ctx);
+      !res) {
     return res.error();
   }
```

Comment:
This is fine, but it is an opportunity to modernize this code with `KATANA_CHECKED`.